src/ui/Card.tsx

```diff
@@ -40,8 +40,7 @@ const WINDOW_WIDTH = Dimensions.get('window').width + TOOLBAR_OFFSET;
 
 enum CurrentAction {
   None = 0,
-  SwipingRight,
-  SwipingLeft,
+  Swiping,
```

Comment:
I think I'll probably end up reintroducing separate swiping states to keep track of whether we're swiping horizontally or vertically, and whether we're in a primary or secondary swipe.

Patch LGTM though.